capstone/macros/test_null_value_threshold.sql

```diff
@@ -0,0 +1,16 @@
+{% test null_value_threshold(model, column_name, threshold = 0.02) %}
```

Comment:
just to clarify: this 0.02 will be used if the threshold field is not passed when calling the test. This is a good thing to do in certain cases but just wanted to clarify that this will act as a default / is worth mentioning in your comment on line 5